themes/src/main/resources/theme/base/login/resources/js/webauthnAuthenticate.js

```diff
@@ -0,0 +1,72 @@
+import { base64url } from "../../../common/keycloak/node_modules/rfc4648/lib/rfc4648.js";
+
+window.authenticateByWebAuthn = (isUserIdentified, challenge, userVerification, rpId, createTimeout, errmsg) => {
+    if (!isUserIdentified) {
+        doAuthenticate([], challenge, userVerification, rpId, createTimeout, errmsg);
+        return;
+    }
+    checkAllowCredentials(challenge, userVerification, rpId, createTimeout, errmsg);
+}
+
+function checkAllowCredentials(challenge, userVerification, rpId, createTimeout, errmsg) {
+    let allowCredentials = [];
```

Comment:
Since this variable is never re-assigned it can be a constant.
```suggestion
    const allowCredentials = [];
```

Reply:
I fixed the codes as you suggested.